jupyterlab/staging/package.json

```diff
@@ -9,6 +9,7 @@
     "build:stats": "webpack --profile --json > stats.json",
     "clean": "rimraf build",
     "prepublishOnly": "npm run build",
+    "postinstall": "yarn-deduplicate yarn.lock",
```

Comment:
These changes should also go in `dev_mode/package.json` since that file is copied here as part of the release process.

Reply:
Thanks, I just pushed the changes!